### PR TITLE
Documentation: How to Configure Facebook Login

### PR DESCRIPTION
**Pull request by tobiasmuehl**
Better version of PR #626

### CODE REVIEW

**Comment by koistya** [comment]
@tobiasmuehl thanks for the PR! I see there is a new npm dependency in the commit, is it on purpose?

File: package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "normalize.css": "4.1.1",
     "passport": "0.3.2",
     "passport-facebook": "2.1.0",
+    "postgraphql": "^1.2.0",
```
Comment (langpavel) [inline package.json]: Great tip, I really fall in love with [`postgraphql`](https://github.com/calebmer/postgraphql#readme) package, but it does'n belongs here.

Comment (tobiasmuehl) [inline package.json]: The new NPM dependency is not on purpose and only found it's way in the PR
by accident

Pavel Lang <notifications@github.com> schrieb am Mi., 18. Mai 2016, 02:50:

Comment (langpavel) [inline package.json]: Please @tobiasmuehl, remove this.. Thanks!

**Pull request closed without merging**